src/Microsoft.AspNetCore.Http.Connections.Client/Internal/WebSocketsTransport.cs

```diff
@@ -23,6 +23,9 @@ public partial class WebSocketsTransport : ITransport
         private readonly TimeSpan _closeTimeout;
         private volatile bool _aborted;
 
+        private static readonly string XRequestedWithName = "X-Requested-With";
+        private static readonly string XRequestedWithValue = "XMLHttpRequest";
```

Comment:
Hmm, if these aren't shared then making them consts is kind of pointless. Is there somewhere public internal to place them? If not then just have the strings inline.